README.md

```diff
@@ -64,22 +64,26 @@ APNS | ✅ | ❌ | [kylebrowning/APNSwift](https://github.com/kylebrowning/APNSw
 PostgreSQL | ✅ | ❌ | [vapor/postgres-nio](https://github.com/vapor/postgres-nio) | `PostgresNIO` | SSWG community project
 Redis | ✅ | ❌ | [mordil/swift-redi-stack](https://gitlab.com/Mordil/swift-redi-stack) | `RediStack` | SSWG community project
 
-### Supported Platforms
+### Supported Versions
 
-SwiftNIO aims to support all of the platforms where Swift is supported. Currently, it is developed and tested on macOS and Linux, and is known to support the following operating system versions:
+### SwiftNIO 2
+This is the current version of SwiftNIO and will be supported for the forseeable future.
 
-* Ubuntu 18.04+
-* macOS 10.9+, iOS 7+; (macOS 10.14+, iOS 12+, tvOS 12+ or watchOS 6+ with [swift-nio-transport-services][repo-nio-transport-services])
+The latest released SwiftNIO 2 version supports Swift 5.2+. NIO 2.29.0 and older support Swift 5.0+.
 
-### Swift versions
+### SwiftNIO 1
+SwiftNIO 1 is considered end of life - it is strongly recommended that you move to a newer version.  The Core NIO team does not actively work on this version.  No new features will be added to this version but PRs which fix bugs or security vulnerabilities will be accepted until the end of May 2022.
```

Comment:
Why not just say "move to NIO 2" instead of "a newer version".

Reply:
I'm prepping for NIO3